src/utils/defer-execution.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Executes the given function in the next run loop using a standard timeout.
+ * Can be useful in the React lifecycle if execution is desired after the
+ * current update.
+ * @param {Function} action The function to execute.
+ * @returns The timeout identifier that can be used to cancel the timeout, if
+ * necessary.
+ */
+function deferExecution(action) {
+  return setTimeout(action, 0);
```

Comment:
I got tired of having a bunch of setTimeouts throughout the code. Hopefully this makes the intention more immediately clear. Naming suggestions welcome.